db/migrations/20250107150808_AddModuleStreams.up.sql

```diff
@@ -45,4 +45,6 @@ ALTER TABLE ONLY module_streams
 DROP CONSTRAINT IF EXISTS fk_module_streams_uniq,
 ADD CONSTRAINT fk_module_streams_uniq UNIQUE (hash_value);
 
+UPDATE repositories SET repomd_checksum = '' where public = true;
```

Comment:
this works, so for non public repos the user may have to recreate the repo? This is probably an edge so maybe it's not a big deal

Reply:
I believe force introspecting the repo should suffice if needed.

In reality, the only 3rd party repo i know that has modules is nvidia's which gets regular updates, so by the time image builder actually support modules, i imagine it would be refreshed.  Or we could manually force its introspection.

Reply:
I think force introspecting only forces the task to run, but the task itself will still exit early. But it sounds like this isn't a realistic issue so I agree it should be fine